### ISSUE TITLE
A case where finetuna fails to reach a reasonable minimum

### ISSUE THREAD

**Issue by gauthierlab** [ulissigroup/finetuna#50]
Hello again! I've found a couple of cases where finetuna fails to reach a reasonable minimum. From my very brief use of it, the common theme seems to be weak binding metals with adsorbates that don't quite bind. It seems like finetuna can fail to localize a minimum with the molecule just above the surface and instead the molecule can float up through the periodic image leading to surfaces potentially blowing up.

I don't know how useful this is, but I've also included a zip file with one such case, *N2H on Ag (111). I'm running a regular VASP optimization to see how the cg optimizer handles this and will update when it's finished.
[to_zulissi.zip](https://github.com/ulissigroup/finetuna/files/9647022/to_zulissi.zip)

**Comment by 12Chao** [comment]
I had a similar issue with CO2 on Cu(111) relaxation when I was testing for PR https://github.com/ulissigroup/finetuna/pull/51. It ended up with a structure being predicted by Finetuna cannot be calculated by the parent calculator, and thus causing a crash  to the parent calculator. 

For CO2 on Cu(111) relaxation calcualtion, it starts with a structure like this
![image](https://user-images.githubusercontent.com/44850205/192354404-dc163c5b-e332-41a1-99a5-fafd00784e5b.png)
and the parent calculator failed at a structure like this 
![image](https://user-images.githubusercontent.com/44850205/192354644-4293772d-a08e-406a-8075-ef333491804c.png)

However, it worked fine for another system (CH3 on Cu(111)) I tested on. The script I used to run the calculation is described in PR https://github.com/ulissigroup/finetuna/pull/51. The input and output trajectory, and log files are attached here. Any answers on what is causing this would be really appreciated. Thanks!
[finetuna_qe_test_gpu.zip](https://github.com/ulissigroup/finetuna/files/9648832/finetuna_qe_test_gpu.zip)

**Comment by zulissi** [comment]
Yes, we've noticed the same thing but hadn't updated the defaults in the examples yet. If you have a weakly adsorbing situation (especially with dispersion on) or the calculation is very different from the training data (RPBE), querying one point in the beginning is much safer (and not a big difference in total number of calls. We also found query every 30 steps to be much more robust. These are the defaults in finetuna_wrap.py, but not the defaults yet elsewhere.

@gauthierlab try using ASE to write the VASP inputs, then use finetuna_wrap.py (https://github.com/ulissigroup/finetuna/blob/main/finetuna/vasp_wrapper/finetuna_wrap.py) to handle everything. That's what we've been doing for most of our recent calculations.

@gauthierlab @12Chao if using the example scripts try adjusting your learner_params to:

```
	learner_params={
	    "stat_uncertain_tol": 1000000,
	    "dyn_uncertain_tol": 1000000,
	    "dyn_avg_steps": 15,
	    "query_every_n_steps": 30, # 30 instead of 100 steps
	    "num_initial_points": 1, # Always sample the very first point with DFT
	    "initial_points_to_keep": [],
	    "fmax_verify_threshold": 0.03,
	    "tolerance_selection": "min",
	    "partial_fit": True,
	}
```
